docs/running-on-mesos.md

```diff
@@ -207,6 +207,16 @@ The scheduler will start executors round-robin on the offers Mesos
 gives it, but there are no spread guarantees, as Mesos does not
 provide such guarantees on the offer stream.
 
+In this mode spark executors will honour port allocation if such is
```

Comment:
Yeah, probably not worth worrying about unless you have a spare minute... I suppose the docs consistently use US spelling but it's hardly a big deal.